mtgjson5/providers/tcgplayer.py

```diff
@@ -65,6 +65,28 @@ class TCGPlayerProvider(AbstractProvider):
 
     api_version: str = ""
     tcg_to_mtgjson_map: Dict[str, str]
+    language_map: Dict[int, str] = {
+        1: "English",
```

Comment:
Since these are dicts with incremental keys, you could easily replace these lookups.

Now: word = language_map[key_id]
Next: word = language_options[key_id - 1]

Reply:
Change of plans -- i'm gonna push a fix. Realized this is reinventing the wheel